pytorch_lightning/trainer/evaluation_loop.py

```diff
@@ -222,13 +222,13 @@ def reset_test_dataloader(self, *args):
     def reset_val_dataloader(self, *args):
         """Warning: this is just empty shell for code implemented in other class."""
 
-    def _evaluate(self, model: LightningModule, dataloaders, max_batches: int, test_mode: bool = False):
+    def _evaluate(self, model: LightningModule, dataloaders, max_batches: List[int], test_mode: bool = False):
         """Run evaluation code.
 
         Args:
             model: PT model
             dataloaders: list of PT dataloaders
-            max_batches: Scalar
+            max_batches: list of Scalar values
```

Comment:
```suggestion
            max_batches: A list of integers with length of the number of dataloaders. Each entry is the number of batches to process in the corresponding dataloader.
```

Reply:
let's also update the docstring while we're at it:
```
model: the model to evaluate
dataloaders: a list of PyTorch dataloaders
```

Reply:
done